packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -288,7 +320,18 @@ export default function CollabSidebar() {
 			}
 		} );
 
-		return result;
+		const blockCommentIds = getCommentIdsFromBlocks();
+
+		const uniqueIds = [ ...new Set( blockCommentIds.values() ) ];
+
+		const threadIdMap = new Map(
+			result?.map( ( thread ) => [ thread.id, thread ] )
```

Comment:
If it's possible for `result` to not be an array, we should probably just return early?